cmd/allocator/main.go

```diff
@@ -184,25 +190,28 @@ func newServiceHandler(kubeClient kubernetes.Interface, agonesClient versioned.I
 		logger.WithError(err).Fatal("starting allocator failed.")
 	}
 
+	h.certMutex.Lock()
```

Comment:
super nit: this could go right above line 198 so that the function call isn't done holding the lock (this makes it a lot clearer what is necessary to lock, which is just r/w access to `h.caCertPool`. Same below for `h.tlsCert`.

Reply:
Fixed! I agree it would make things more clear for someone reading the code.